README.pg_sphere

```diff
@@ -11,7 +11,7 @@ It provides:
 
 This is an R-Tree implementation using GiST for spherical objects like
```

Comment:
I propose to change R-**T**ree to R-**t**ree.

Reply:
Yes, please.